.github/workflows/build-test.yml

```diff
@@ -0,0 +1,45 @@
+name: Snyk Vulnerability Scan
```

Comment:
lets name this build pipeline or something more generic.

Reply:
updated to build pipeline